utils/docker/Dockerfile.centos.8

```diff
@@ -13,28 +13,35 @@ LABEL maintainer="daos@daos.groups.io"
 ARG CB0
 
 ARG REPO_URL
-# Need to clean this up later
-# This is largely the same as the centos.7 code, however this sets priority for
-# the new repo, and leaves the default ones enabled as there is some metadata
-# that is not available locally.
-RUN if [ -n "$REPO_URL" ]; then						\
-      cversion=$(cut -d ' ' -f4 /etc/centos-release);			\
-      version=${cversion%.*} ;						\
-      MY_REPO="${REPO_URL}repository/centos-${version}-x86_64-group/";	\
-      MY_NAME="${MY_REPO#*//}";						\
-      MY_NAME="${MY_NAME//\//_}";					\
+ARG REPO_EL8=""
+
+# Need to do this before switching to a alternate local repo
+RUN dnf clean all &&                                            \
+    dnf -y install epel-release dnf-plugins-core dnf-utils &&   \
+    dnf config-manager --set-enabled powertools
+
+# If a local repository is supplied, then we should use it
+# and disable using the distro repositories.
+RUN if [ -n "$REPO_URL" ]; then                                 \
+      MY_REPO="${REPO_URL}${REPO_EL8}/";                        \
+      MY_NAME="${MY_REPO#*//}";                                 \
+      MY_NAME="${MY_NAME//\//_}";                               \
       echo -e "[${MY_NAME}]\n\
 name=created from ${MY_REPO}\n\
 baseurl=${MY_REPO}\n\
 enabled=1\n\
-priority=1\n\
-gpgcheck=False\n" >> /etc/yum.repos.d/local-centos-group.repo;		\
-    fi
+repo_gpgcheck=0\n\
+gpgcheck=0\n" >> /etc/yum.repos.d/local-centos-group.repo;      \
```

Comment:
Same issues as with the EL7 file and GPG checking.  I won't comment on all of them in this file since they are likely the same as the EL7 issues.

In fact probably every comment about the EL7 file, apart from the mixed `yum`/`dnf` issues applies to this file.